box-content-sample/src/main/java/com/box/androidsdk/sample/MainActivity.java

```diff
@@ -68,8 +68,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
     private void initialize() {
         mAdapter.clear();
-
-        mSession = new BoxSession(this, null);
+        mSession = new BoxSession(this);
```

Comment:
If this is how we recommend the BoxSession be used (so that the login is not always reset) - we should probably use it in our samples.